### ISSUE TITLE
Slope Blur Update

### ISSUE THREAD

**Issue by MitchWoodin** [RodZill4/material-maker#664]
Hi,

I've recently been messing around with Material Maker as a substance alternative. I was checking out the slope blur and I think the implementation is at least partly wrong. (At least as far as I'm aware)
At the moment the node is doing a directional blur towards the white part of a greyscale texture. However the way it should be behaving is by warping in both directions and adding those all together. There is a very good video explaining how the slope blur node works in substance here:
https://www.youtube.com/watch?v=H0c1SffwllY&t=308s

I think making this change in Maker would be a good improvement to make. Currently the slope blur node isn't producing the result exactly how I would expect them to when compared to Substance.

**Comment by RodZill4** [comment]
Material Maker is not a SD clone (I absolutely never used SD), and its slope blur node is not supposed to have the exact same behavior (it's a gaussian blur guided by the slope of its input).
It is probably possible to create a node that behaves the same, but "fixing" that node would just break compatibility, which won't happen (does not mean the new node could not be named "Slope Blur 2" or something).

**Comment by MitchWoodin** [comment]
Yep, I think the MultiWarp is actually doing what I'm seeing in Substance!

**Comment by RodZill4** [comment]
Yeah, multiwarp is closer to SD's Slope Blur.
